src/components/pages/HomePage.tsx

```diff
@@ -1,7 +1,76 @@
-import HomeMessage from '../layout/HomeMessage';
+import { useState } from 'react';
+
+import { Alert, Box, Button, Stack, Typography } from '@mui/material';
+
+import { SearchInput } from '@graasp/ui';
+
+import { ITEM_PAGE_SIZE } from '@/config/constants';
+import { useAnalyticsTranslation } from '@/config/i18n';
+import { hooks } from '@/config/queryClient';
+
+import ItemLink from '../common/ItemLink';
+import ItemLoaderSkelton from '../common/ItemLoaderSkeleton';
 
 const HomePage = (): JSX.Element => {
-  return <HomeMessage />;
+  const { t } = useAnalyticsTranslation();
+
+  const [page, setPage] = useState(1);
+  const [searchQuery, setSearchQuery] = useState('');
+  const { data: accessibleItems, isLoading } = hooks.useAccessibleItems(
+    {
+      name: searchQuery,
+    },
+    // get items cumulative
+    { pageSize: page * ITEM_PAGE_SIZE },
+  );
+
+  if (accessibleItems?.data) {
+    return (
+      <Stack padding={3} spacing={2}>
```

Comment:
I think it would be interesting to add a `Container` https://mui.com/material-ui/react-container/ it would add margin on the right and left automatically on big screens!